src/views/ProductView.vue

```diff
@@ -33,45 +33,53 @@
                 </v-sheet>
                 <v-card class="align-self-start" elevation="4">
                     <v-card-item>
-                        <v-card-title>{{ internalName }}</v-card-title>
+                        <v-card-title>{{
+                            product?.product.internalName
+                        }}</v-card-title>
                         <v-card-subtitle>{{
-                            productVariantName
+                            productVariant?.currentVersion.name
                         }}</v-card-subtitle>
                     </v-card-item>
                     <v-card-text>
                         <div class="d-flex flex-wrap ga-2">
-                            <v-chip>Clothing</v-chip>
-                            <v-chip>T-Shirts</v-chip>
+                            <v-chip>Dummy Clothing</v-chip>
```

Comment:
Are here still dummy values required?

Reply:
I think at that point there were still no categories available from the service. Adding the categories to the frontend has to happen next, of course.

Reply:
Imo, everything related to the categories should be added via a dedicated PR in the near future.